app/controllers/api/avatars.php

```diff
@@ -49,11 +53,92 @@
     $response
         ->addHeader('Expires', \date('D, d M Y H:i:s', \time() + 60 * 60 * 24 * 30) . ' GMT')
         ->setContentType('image/png')
-        ->file($data)
-    ;
+        ->file($data);
     unset($image);
 };
 
+$getUserGitHub = function (string $userId, Document $project, Database $dbForProject, Database $dbForConsole) {
+    try {
+        $user = Authorization::skip(fn () => $dbForConsole->getDocument('users', $userId));
+
+        $sessions = $user->getAttribute('sessions', []);
+        $session = $sessions[0] ?? new Document();
```

Comment:
shouldn't it throw an exception if there is no session?

Reply:
It will end up throwing exception PROJECT_PROVIDER_UNSUPPORTED a little below.